experimental/include/experimental/Transforms/LSQSizing/LSQSizingSupport.h

```diff
@@ -0,0 +1,164 @@
+#include "dynamatic/Support/TimingModels.h"
+#include "dynamatic/Transforms/BufferPlacement/CFDFC.h"
+#include <list>
+#include <mlir/IR/Operation.h>
+#include <optional>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#ifndef DYNAMATIC_TRANSFORMS_LSQSIZING_LSQSIZINGSUPPORT_H
+#define DYNAMATIC_TRANSFORMS_LSQSIZING_LSQSIZINGSUPPORT_H
+
+namespace dynamatic {
+namespace experimental {
+namespace lsqsizing {
+
+// Define a structure for a graph node
+struct AdjListNode {
+  int latency;                 // Latency of the operation
```

Comment:
Do not put in-line comments on lines which also contain code, put the comment above instead.

```cpp
/// Latency of the operation.
int latency;
```

The former has two disadvantaged:

1. It tends to create weird indentation with clang-format when wrapping a comment over multiple lines, hurting readability.
2. When commenting type members like here, I don't think most IDEs will pick up that the comment is associated to the member; users will not see the documentation when hovering over the member.